extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/linkHandler/PeertubeSearchQueryHandlerFactory.java

```diff
@@ -20,7 +22,12 @@ public static PeertubeSearchQueryHandlerFactory getInstance() {
 
     @Override
     public String getUrl(String searchString, List<String> contentFilters, String sortFilter) throws ParsingException {
-        String baseUrl = ServiceList.PeerTube.getBaseUrl();
+        String baseUrl;
+        if (contentFilters.size() > 0 && contentFilters.get(0).startsWith("sepia_")) {
```

Comment:
same question as above

Reply:
because if we implement other search filters (e.g. channels), it will not change, the baseUrl with sepia filters will still be sepiasearch.org (by default, unless another baseUrl for another index is provided)